### DISCUSSION TITLE
Can't get LED brightness control to work

### DISCUSSION THREAD

**Discussion by squaretones** [shanteacontrols/OpenDeck#255] [Q&A] [answered]
Hi, I'm using the OpenDeck M board and have a single color red LED plugged into R1 and C1 (the LED 0 slot). I have tried configuring it to both MIDI in / Note (multi value) and MIDI in / CC (multi value) and I can't get either of them to control the brightness. No matter what note velocity or CC value I send, the LED has the same brightness. Any idea on what I could be doing wrong?

To clarify, I understand based on the documentation that if the LED is configured to MIDI in / Note (multi value) and the Activation ID is set to 0, a MIDI note 0 with velocity 124 will produce 25% brightness and MIDI note 0 with velocity 127 will produce 100% brightness. That is not working for me. I just get one brightness, regardless of velocity.

Thanks

**Comment by paradajz** [comment, marked as answer]
The new OpenDeck M boards use the same hardware as the L boards for LEDs and that hardware doesn't support brightness control. Sadly I forgot to mention this in the documentation for the M boards. Added just now.

https://github.com/shanteacontrols/OpenDeck/wiki/Official-OpenDeck-M-boards#single-color-leds
https://github.com/shanteacontrols/OpenDeck/wiki/Official-OpenDeck-L-boards#single-color-leds

**Reply by squaretones** [reply]
Understood. Thanks for letting me know.
